01-introduction.Rmd

```diff
@@ -66,16 +62,23 @@ By teaching you its foundations and underlying structure, we aim to empower you
 ## Why Geocomputation with R?
 
 Early geographers used a variety of tools including rulers, compasses and sextants to advance knowledge about the world. 
+However, until John Harrison invented the marine chronometer in the 18th century it had been impossible to determine the exact longitude at sea ('the longitude problem').
+Prior to his invention ships followed for centuries a line of constant latitude making each journey much longer, more expensive and often also more dangerous.
+Nowadays this seems unimaginable with every smarthphone having a GPS receiver at its disposal.
 <!--nowadays part-->
-Nowadays, we use satellites, GPS, and multitudes of other sensors, and especially computational power to gain insights.
-The latter is of special importance with regard to the ever increasing amount of (geo-)data.
+And there are a multitude of other sensors measuring the world in real-time (satellites, radar, autonomous cars, citizens, etc.).
 For instance, an autonomous car is believed to create one terrabyte of data on a daily base.
-We need software and related tools to extract the most interesting patterns of such data.
-In this book we treat R as a 'tool for the trade' when it comes to analyzing such data.
-However, data management and storing is better left to other software such as dedicated (geo-)database software.
-R is an object-oriented, [functional programming language](http://adv-r.had.co.nz/Functional-programming.html) [@wickham_advanced_2014].
-Furthermore, R was specifically designed as an interactive interface to other software [@chambers_extending_2016]. 
-This also includes many 'bridges' to a treasure trove of GIS software, libraries and functions.
+Equally, earth observation data (satellite imagery) has become so big that it is impossible to analyze the corresponding data with a single computer (see [http://r-spatial.org/2016/11/29/openeo.html](http://r-spatial.org/2016/11/29/openeo.html)).
+Hence, we need computational power, software and related tools to handle and extract the most interesting patterns of this ever-increasing amount of (geo-)data.
+In this book we treat R as a 'tool for the trade' when it comes to analyzing (subsets of) such data.
```

Comment:
I mean it's what people use for a specific type of work where the work is the focus not the tool.